### DISCUSSION TITLE
Update Azure EJBCA EE using redeploy in Azure

### DISCUSSION THREAD

**Discussion by bergstiefelbene** [Keyfactor/ejbca-ce#271] [Q&A] [answered]
Hi,

we are using a three node azure ejbca cluster with internal MariaDB which is synced. To update the older version of the Node01, and Node02 we got the information that we could use the Azure Redeploy option to update the Azure client version from ejbca_ee_7_10_0_1 to ejbca_ee_7_11_0. Today we tried that but after the VM was carried to another Azure host and all the data as well, the version did not changed. 
In the past we just used the downloaded zip. Deployed it manually in the /opt directory with relink to the newer version. Restart and go.
Did you have ever worked with that redeploy and update possibility like the ejbca support told us?

**Comment by primetomas** [comment]
Since no-one else stepped forward... Guess that is EE specific/Marketplace? I assume if support told it it is tested, a problem with it should be reported back and they'll investigate it.

**Reply by primetomas** [reply]
I.e., I haven't worked with that myself.

**Comment by bergstiefelbene** [comment, marked as answer]
Thanks for the answer. We just update like in the past without problem. The redeploy process is no an option in the moment.
